src/components/utils/hooks/useGetMessages.tsx

```diff
@@ -11,13 +12,20 @@ const useGetMessages = (conversation?: Conversation) => {
       return;
     }
     const loadMessages = async () => {
-      const newMessages = await conversation.messages();
-      messages.set(conversation.peerAddress, newMessages);
+      console.log(endTime);
+      const newMessages = await conversation.messages({
+        direction: SortDirection.SORT_DIRECTION_DESCENDING,
+        limit: 5,
```

Comment:
Unsure if this is just for testing, but I think we can likely bump this page size up a bit -- maybe something like last 20 messages?

Reply:
20 seems like a reasonable number to start. We can play with it using some large conversations and see how it feels.

Reply:
Also @bhavya2611 @neekolas , is it possible to use `await conversation.messagesPaginated({ pageSize: 20 })` here so we can page thru messages without having to query by end time?

Reply:
I was trying to figure out a nice way to do that. It's a little messy trying to pause an async generator. You have to await some Promise in the loop, and have that promise get resolved externally and replaced with a new promise before the next loop. Otherwise it will just page until the end. It's possible, but not pretty.